client/boot/app.js

```diff
@@ -0,0 +1,60 @@
+// Initialize localStorage polyfill before any dependencies are loaded
+require( 'lib/local-storage' )();
+if ( process.env.NODE_ENV === 'development' ) {
+	require( 'lib/wrap-es6-functions' )();
```

Comment:
fascinating. this transpiles to 
```
// Initialize localStorage polyfill before any dependencies are loaded
	__webpack_require__(4672168639826360)();
	if (false) {
		require('lib/wrap-es6-functions')();
	}
```

notice the `require` didn't get transformed to a `__webpack_require__`. Also the functions in `'lib/wrap-es6-functions'` are not present in the build bundle. 

cool, I didn't think this actually worked.